kss/builder/decanter/index.twig

```diff
@@ -273,6 +273,8 @@
   });
 </script>
 {{ scripts|raw }}
+<footer class="page-footer">
```

Comment:
```suggestion
<footer class="site-footer">
```
I forgot to change the class name for the kss twig after I changed it in the mixin.